modules/installation-configuration-parameters.adoc

```diff
@@ -1938,71 +1938,47 @@ Additional VMware vSphere configuration parameters are described in the followin
 |====
 |Parameter|Description|Values
 
-l|platform:
-    vsphere
-      apiVIPs
+|`apiVIPs`
```

Comment:
These should be the full paths, I think? can refer back to the PRs that changed them to `l` for reference, e.g. https://github.com/openshift/openshift-docs/pull/64748/files
```suggestion
|`platform.vsphere.apiVIPs`
```

Reply:
![Screenshot from 2024-03-12 17-06-58](https://github.com/openshift/openshift-docs/assets/57954076/b9b40b16-e0ce-42d0-94c8-1016c87b2bc3)

`platform.vsphere` should be assumed as its the initial array pattern.

Reply:
I agree, and lean a bit towards 2 for readability but 1 for making extra sure - so I guess I don't strongly favor one over the other

Reply:
Thanks, @bscott-rh and @jeana-redhat . I added the following the sentence to the target tables:

> The `platform.vsphere` parameter prepends each parameter listed in the table.

I can wrap this sentence in an admonition block, but maybe it will distract from the initial intro sentence. WDYT?

Tables updated:

* Additional VMware vSphere configuration parameters
* Deprecated VMware vSphere configuration parameters
* Optional VMware vSphere machine pool configuration parameters

Reply:
That works for me

Reply:
LGTM